lib/hydra/works/models/concerns/generic_file_behavior.rb

```diff
@@ -20,6 +20,42 @@ module GenericFileBehavior
       include Hydra::Works::GenericFile::Derivatives
       include Hydra::Works::GenericFile::MimeTypes
       include Hydra::Works::GenericFile::VersionedContent
+      include Hydra::Works::BlockChildObjects
+
+      filters_association :members, as: :child_generic_files, condition: :works_generic_file?
```

Comment:
Opened an issue to discuss this further.  The PR does not change the behaviors relative to the current code at master.

https://github.com/projecthydra-labs/hydra-works/issues/165